contrib/storage-hive/core/src/test/java/org/apache/drill/exec/hive/complex_types/TestHiveArrays.java

```diff
@@ -59,6 +60,7 @@
 
   @BeforeClass
   public static void setUp() throws Exception {
+    assumeTrue("Skipping tests since Hive supports only JDK 8.", HiveTestUtilities.supportedJavaVersion());
```

Comment:
Maybe its better to create parent class for all Hive tests and add check there so we don't have to repeat the check?